pydantic/fields.py

```diff
@@ -748,7 +746,7 @@ class ComputedFieldInfo:
 
     decorator_repr: typing.ClassVar[str] = '@computed_field'
     wrapped_property: property
-    json_return_type: _core_schema.JsonReturnTypes | None
+    return_type: type[Any]
```

Comment:
Is it okay that we use `Any` below but `type[Any]` here? I can understand why you might do this, but just want to make sure it was intentional.

Reply:
My reasoning was that `type[Any]` is technically incompatible if you try to assign `int | str`, but internally we pretty much want to treat it as a type, so `type[Any]` is more semantically meaningful